dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -192,6 +214,12 @@ export const SignInGateSelector = ({
 				})
 				.then(setCanShowGate);
 		}
+		// eslint-disable-next-line @typescript-eslint/no-floating-promises
+		getSwitches().then((switches) => {
+			if (switches.personaliseSignInAfterCheckout) {
+				setPersonaliseSwitch(switches.personaliseSignInAfterCheckout);
+			} else setPersonaliseSwitch(false);
```

Comment:
Is the `useEffect` method an appropriate place to make an async call here? I'm using a statehook to store the result of this call.

Reply:
Checked with Olly and this looks good